app/models/forecast.rb

```diff
@@ -1,15 +1,68 @@
 require 'forecast_io'
+require 'openssl' #evil ssl removing hack for windows since windows doesn't have a root store. I am not going to lie, this is sleezy.
+
+#Hack: http://stackoverflow.com/questions/170956/how-can-i-find-which-operating-system-my-ruby-program-is-running-on Just testing.
+def isWindows?
+  (/cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM) != nil
+end
+
+if isWindows? then
+  OpenSSL::SSL::VERIFY_PEER = OpenSSL::SSL::VERIFY_NONE
+  #that's evil code. It just disables cert verification on my machine since I am windows and windows doesn't work with ruby on ssl (in the amount of time I have to debug this).
+end
```

Comment:
@derekriemer if we still need this then we could probably just move it to the Rake file and leave it there.